web/sites/bnf/settings.php

```diff
@@ -8,6 +8,9 @@
  * settings.lagoon.php that contains Lagoon-specific settings.
  */
 
+$config['varnish_purger.settings.30cd45a1b1']['hostname'] = 'bnfvarnish';
+$config['varnish_purger.settings.65fc931232']['hostname'] = 'bnfvarnish';
```

Comment:
Where are you getting these values from? (30cd45a1b1 / 65fc931232)

Reply:
Ah, I forgot a comment.

Reply:
Can these values change? I'm a bit worried that one day, this will break, because of a re-export

Can we set this with a hook instead, or something?

Reply:
They change if someone deletes the purgers and create new. Which is not supposed to happen.